targets/csharp/make.js

```diff
@@ -29,7 +29,6 @@ exports.makeCombinedAPI = function (apis, sourceDir, apiOutputDir) {
     };
 
     templatizeTree(locals, path.resolve(sourceDir, "source"), rootOutputDir);
-    templatizeTree(locals, path.resolve(sourceDir, "UnittestRunner"), path.resolve(apiOutputDir, "UnittestRunner")); // Copy the actual unittest project in the CombinedAPI
```

Comment:
Will the source dir templatizeTree pick up tests for us now?

Reply:
Yes, I just moved the same files into the "source" subfolder, one line up.